docs/internals/contributing/writing-code/submitting-patches.txt

```diff
@@ -236,11 +236,12 @@ Finally, there are a couple of updates to Django's documentation to make:
    the "Features deprecated in A.B" heading.
 
 #) Add an entry in the deprecation timeline (``docs/internals/deprecation.txt``)
-   under the ``A.B+2`` version describing what code will be removed.
+   under the ``A.B+2`` (or B+3 for features deprecated in the version after an
```

Comment:
Actually now that I write out the three possibilities I see that it's never actually `A.B+2`. Which makes sense of course; a deprecated feature should never be removed in the same major release series it was deprecated in, since we're doing semver(ish).